README.md

```diff
@@ -30,14 +30,30 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+    - .map will return a value and will never update the original array unless stated. .forEach will, by default, return undefined and will update the original array.
```

Comment:
One more thing to add - map doesn't mutate the original array, because it returns a new array.